flexget/plugins/filter/series.py

```diff
@@ -1001,6 +1001,31 @@ def set_series_begin(series, ep_id):
     series.begin = episode
 
 
+def add_series_entity(session, series, identifier, quality=None, downloaded=True):
+    """
+    Adds entity identified by `identifier` to series `name` in database.
+
+    :param series: Series in database to add entity to.
+    :param identifier: Series identifier to be added.
+    :param quality: If supplied, this will override the quality from the series parser.
+    """
+    name_to_parse = '{} {}'.format(series.name, identifier)
+    if quality:
+        name_to_parse = '{} {} {}'.format(series.name, identifier, quality)
```

Comment:
You mean just pass quality straight to `store_parser` instead of appending it to the string to be parsed?